sqlutil/percentile.go

```diff
@@ -45,3 +47,26 @@ func FormatPercentile(f float64) string {
 	metaKey = "p" + metaKey
 	return metaKey
 }
+
+type P struct {
+	Name  string
+	Value float64
+}
+
+func PercentileMetrics(metrics []string) ([]P, error) {
```

Comment:
Would be better to have test case for it.

Reply:
Tests pushed.